src/Module/Search/Index.php

```diff
@@ -151,30 +152,18 @@ public static function content(array $parameters = [])
 		if ($tag) {
 			Logger::info('Start tag search.', ['q' => $search]);
 			$uriids = Tag::getURIIdListByTag($search, local_user(), $pager->getStart(), $pager->getItemsPerPage());
-
-			if (!empty($uriids)) {
-				$params = ['order' => ['id' => true], 'group_by' => ['uri-id']];
-				$items = Item::selectForUser(local_user(), [], ['uri-id' => $uriids], $params);
-				$r = Item::inArray($items);
-				$count = Tag::countByTag($search, local_user());
-			} else {
-				$count = 0;
-			}
 		} else {
 			Logger::info('Start fulltext search.', ['q' => $search]);
+			$uriids = ItemContent::getURIIdListBySearch($search, local_user(), $pager->getStart(), $pager->getItemsPerPage());
```

Comment:
You need a total count for this case as well.

Reply:
Done